src/sflow_plugin/sflow_collector.cpp

```diff
@@ -86,6 +89,10 @@ void start_sflow_collection(process_packet_pointer func_ptr) {
         interface_for_binding = configuration_map["sflow_host"];
     }
 
+    if (configuration_map.count("sflow_qinq_ethertype") != 0) {
+        sflow_qinq_ethertype = convert_hex_as_string_to_uint(configuration_map["sflow_qinq_ethertype"]);
```

Comment:
Need error checking. If we cannot parse it, just emit error and disable processing for sFlow flags or use default value.

Reply:
I added the processing flag setting to the disabled state if the value could not be parsed. Also, a warning message is added to the log file.